torch/csrc/autograd/python_function.h

```diff
@@ -94,6 +94,11 @@ struct THPFunction {
     std::vector<bool> is_variable_input;
     char has_freed_buffers;
 
+    PyObject *saved_for_forward;
+    // In jvp the 'saved_tensor' attribute return tensors saved with saved_for_forward
+    // In vjp (or backward) the 'saved_tensor' attribute unpacks and returns tensors
+    // saved with saved_for_backward
+    bool is_saved_tensor_for_backward = false;
```

Comment:
I am still not convinced that this flag is 100% necessary. And if it is I am not convinced by the name.

Reply:
Removed!